skillmap/src/lib/indexedDBWrapper.ts

```diff
@@ -0,0 +1,125 @@
+// Duplicated from pxtlib/browserutils.ts
+// TODO: Move to common location
+
+
+// IndexedDB wrapper class
+export type IDBUpgradeHandler = (ev: IDBVersionChangeEvent, request: IDBRequest) => void;
+
+export class IDBWrapper {
```

Comment:
nit: since browserutils is being copied over now, this file can go away, right?

Reply:
i think we're doing a separate pass to dedup everything (there are a couple functions in the skillmap/lib/browserutils that also need to be removed)